core/src/main/java/com/epam/esm/entities/Entity.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.esm.entities;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * Stores ID field for data from "gift_certificate" and "tags" tables
+ */
+public abstract class Entity implements Serializable {
```

Comment:
@KukharchukDaniil Certificates and Tags have name field. Why don't you use this field here?

Reply:
fixed